src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+experience and this is called "delegation".
+
+<figure class="w-figure" style="width:300px; margin:auto;">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Checkout flow with a web-based payment app.
+  </figcaption>
+</figure>
+
+Web-based payment apps can declare their support for delegation using the web
+app manifest's `payment.supported_delegations` field or through JavaScript.
+
+To let the browser know that the payment app accepts a delegation, use
+[`.enableDelegations()`](https://w3c.github.io/payment-handler/#enabledelegations-method).
+
+{% Label %}[payment handler] payment app's landing page{% endLabel %}
+
+```js
+...
+ await registration.paymentManager.enableDelegations([
+  'shippingAddress', 'payerName'
+]);
+...
+```
+
+You can declare supported delegation options as an array of strings:
+
+* `shippingAddress`: The payment handler can provide a shipping address.
+* `payerName`: The payment handler can provide the payer's name.
+* `payerPhone`: The payment handler can provide the payer's phone number.
+* `payerEmail`: The payment handler can provide the payer's address.
+
+## Debugging a web-based payment app
+
+When developing a web-based payment app frontend, you'll probably jump between
+merchant perspective and payment app perspective. The following debugging tips
+will help your developing experience on Chrome.
+
+### Developing on a local server
+
+Which server do you use for development? Many developers tend to use localhost
+or a company-internal server environment which can be challenging because
+powerful features in the browser tend to require a secure environment (HTTPS)
+and a valid certificate. The Payment Request API and the Payment Handler API are
+no exception and localhosts or company-internal servers usually don't come with
+a valid certificate.
+
+The good news is some browsers, including Chrome, exempt certificates for
+`http://localhost` by default. Also in Chrome, you can exempt the certificate
+requirement by launching a Chrome instance. For example, to exempt the
+requirement from `http://*.corp.company.com`, use the following flags:
+
+* [`--ignore-certificate-errors`](https://chromiumdash.appspot.com/commit/988b56b519836f3d3d94f145ba3554a0c0a7d0a8)
+* [`--unsafely-treat-insecure-origin-as-secure=http://*.corp.company.com`](https://chromiumdash.appspot.com/commit/77a7e1a65d14072149ec4420a0ab523586011d8a)
+
+### Port forwarding a local server
+
+You can port forward the local web server to an Android device using Chrome's
+DevTools and test how it works from a mobile browser. To learn how to do it,
+check out [Access Local
+Servers](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/local-server).
```

Comment:
FYI I haven't looked at that guide in a long time and I'm pretty sure some developers complained that the workflow didn't work. Have you tried it yourself?

Reply:
No, I didn't :/

Reply:
I know this is extra work but can you try it out? I would do it myself but you're way more familiar with these APIs and should be able to do it faster. It's really disrespectful of our users if we recommend debugging workflows without knowing whether they actually work.

Reply:
https://github.com/google/WebFundamentals/pull/8876